src/games/strategy/triplea/ui/screen/IDrawable.java

```diff
@@ -11,7 +11,7 @@
 import games.strategy.triplea.image.MapImage;
```

Comment:
The changes here are results of changes in MapData.
Note:
` final Optional<Point> namePlace = mapData.getNamePlacementPoint(territory);`
The return type is now optional. The changes here are strictly to deal with that.